opencv_ai/splice/component.h

```diff
@@ -0,0 +1,60 @@
+#ifndef OPENCV_AI_SPLICE_COMPONENT_H_
+#define OPENCV_AI_SPLICE_COMPONENT_H_
+
+#include <boost/optional.hpp>
+#include <boost/regex.hpp>
+
+#include <glog/logging.h>
+
+#include <opencv_ai/splice/context.h>
+
+namespace opencv_ai {
```

Comment:
And if we're going to follow the google/boost umbrella namespace pattern, should we go opencv::ai?